dpsim/include/dpsim/DAESolver.h

```diff
@@ -76,6 +76,7 @@ class DAESolver : public Solver {
   /// Deallocate all memory
   ~DAESolver();
   /// Initialize Components & Nodes with inital values
+  void initialize() override;
   void initialize(Real t0);
```

Comment:
There is a conflict due to the use of `Real t0` in here...

https://github.com/sogno-platform/dpsim/blob/0f4d17c6dae7efa5b6c27f6c1f1aca767a50aa20/dpsim/src/DAESolver.cpp#L156

I do not know if we could do this differently so it does actually override